lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1865,20 +1865,27 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :dynamic_name => true
         },
         :'application_logging.forwarding.enabled' => {
-          :default => true,
+          :default => false,
           :public => true,
           :type => Boolean,
-          :allowed_from_server => true,
-          :description => 'If `true`, the agent captures log records emitted by your application.'
+          :allowed_from_server => false,
+          :description => 'If `true`, the agent captures log records emitted by this application.'
         },
         :'application_logging.forwarding.max_samples_stored' => {
-          :default => 2000,
+          :default => 10000,
```

Comment:
Question - does the truncation per entry get performed after the sample storage max is reached? Essentially... should we truncate before storing anything in memory?

Reply:
I'm not sure I understand what you mean. Are you talking about whether we sample the logs before hitting the max? Or truncating the messages? Something else?

For sampling, we do not begin sampling until the max is reached, then things are sampled based on their priority. This means that if a customer has 9999 logs within a single harvest cycle, they should have all 9999 reported. 

For log messages, everything should be truncated if it exceeds the max bytesize before it is stored in memory.

Reply:
Ok, cool. Your last statement that everything is truncated BEFORE being stored in memory is what I was wondering about. Perfect!

I was worried how much memory would be taken up by 10,000 untruncated messages.